src/main/java/uk/gov/hmcts/darts/common/entity/EventEntity.java

```diff
@@ -23,17 +26,15 @@ public class EventEntity {
     @SequenceGenerator(name = "moj_eve_gen", sequenceName = "moj_eve_seq", allocationSize = 1)
     private Integer id;
 
-    @Column(name = "moj_crt_id")
+    @Column(name = "moj_ctr_id")
```

Comment:
moj_ctr_id is courtroom, courthouse is moj_cth_id

Reply:
Its courtroom we want, the mapping is already specified further down so have just removed it.